test/server/core/abilities/keyword/Saboteur.spec.js

```diff
@@ -27,15 +27,17 @@ describe('Saboteur keyword', function() {
             });
 
             it('a unit with shields, the shields are defeated before the attack', function () {
-                expect(this.player2.findCardByName('shield').location).toBe('ground arena');
                 this.player1.clickCard(this.resourcefulPursuers);
                 this.player1.clickCard(this.wampa);
                 expect(this.resourcefulPursuers.damage).toBe(4);
                 expect(this.echoBaseDefender).toBeInLocation('ground arena');
                 expect(this.wampa.damage).toBe(5);
                 expect(this.wampa).toBeInLocation('ground arena');
+                expect(this.wampa).toHaveExactUpgradeNames(['resilient']);
                 expect(this.player2.findCardByName('shield').location).toBe('outside the game');
```

Comment:
I know I've mentioned it a couple times but I think we need to remove this expectation. I realized the issue is that the shield isn't even in the "outside the game" location anymore, it got removed from that array (effectively all references to it should have been deleted so the GC will come pick it up). The fact that it's even able to be found is a bug, I'm not sure why it's coming up in `findCardByName`.

If you switched this to the new expectation (`.isInLocation()`) it would fail because the location property (outside the game) no longer matches the actual array the card lives in (it doesn't have one).